gnome-extra/chrome-gnome-shell/chrome-gnome-shell-10-r2.ebuild

```diff
@@ -33,6 +33,17 @@ src_configure() {
 	cmake-utils_src_configure
 }
 
+src_install() {
+	cmake-utils_src_install
+
+	if [[ $(get_libdir) != "lib" && "${SYMLINK_LIB}" != yes ]]; then
+		# Workaround www-client/firefox-bin manifests location
+		# Bug: https://bugs.gentoo.org/643522
+		insinto /usr/lib/mozilla/native-messaging-hosts
+		doins "${ED}"/usr/$(get_libdir)/mozilla/native-messaging-hosts/org.gnome.chrome_gnome_shell.json
+	fi
```

Comment:
is there a reason why we can't just install into `/usr/lib` unconditionally?

Reply:
As stated in bug 643522 firefox expect manifest in ```libdir``` while firefox-bin expect it in ```lib``` always